src/retro-c.cpp

```diff
@@ -0,0 +1,476 @@
+#include "retro-c.h"
+
+#include "coreinfo.h"
+#include "data.h"
+#include "emulator.h"
+#include "imageops.h"
+#include "memory.h"
+#include "search.h"
+#include "script.h"
+#include "movie.h"
+#include "movie-bk2.h"
+
+#include <map>
+#include <unordered_map>
+#include <unordered_set>
+
+using namespace Retro;
+
+CEmulator* emulatorCreate(const char* romPath) {
+  Retro::Emulator* emulator = new Retro::Emulator();
```

Comment:
Please match the existing style.

Reply:
Also if you're using namespace Retro you don't need the Retro:: here.

Reply:
I only used `Retro::` to try and match the style used in your Python API implementation. I removed that now, but what else do you refer to when you ask to match the existing style? Is it about renaming `emulator` to `m_re`?